plugin/trino-hive/src/main/java/io/trino/plugin/hive/util/HiveBucketing.java

```diff
@@ -80,6 +80,12 @@ int getBucketHashCode(List<TypeInfo> types, Page page, int position)
             {
                 return HiveBucketingV1.getBucketHashCode(types, page, position);
             }
+
+            @Override
+            public int getBucketHashCode(List<TypeInfo> types, Page page, int[] columns, int position)
```

Comment:
make it `getBucketHashCode(List<TypeInfo> types, Page page, int prefixChannels, int position)` and make `getBucketHashCode(List<TypeInfo> types, Page page, int position)` redirect to this implementation